lib/rubocop/cop/rspec/before_after_all.rb

```diff
@@ -0,0 +1,42 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Check that before/after(:all) isn't being used.
+      # This leads to flaky tests when being split out.
```

Comment:
I'll update the description to make it clearer based on your previous feedback yeah! 💯